# commit f8c3b831ea2af3de9036c3dbebcff27fbfcfeec4
# Author: Matthias Andree <matthias.andree@gmx.de>
# Date:   Fri May 27 01:22:44 2011 +0200
# 
#     Bugfix: don't delete articles when texpire -n -C MessID is run.
#     
#     Found by Paul Brooks.

--- a/texpire.c
+++ b/texpire.c
@@ -1123,7 +1123,9 @@ main(int argc, char **argv)
 	    while(optind < argc) {
 		if (verbose)
 		    printf("Trying to remove %s...\n", argv[optind]);
-		delete_article(argv[optind], "Remove", "Removed", 1);
+		if (!dryrun) {
+		    delete_article(argv[optind], "Remove", "Removed", 1);
+		}
 		optind++;
 	    }
 	    break;
